scripts/st2bootstrap-el7.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+set -eux
```

Comment:
If you need to explicitly exit on some failure, catch exit codes and quit. BTW ubuntu script doesn't do this. Not a bash expert here and I am ears for opinions but status quo is bad in terms of UX.